app/schema.graphql

```diff
@@ -0,0 +1,103 @@
+scalar Datetime
+
+type Author {
```

Comment:
Going to drop some links here to help with discovering how to get info from `git2`.

[Commit.author](https://docs.rs/git2/0.10.1/git2/struct.Commit.html#method.author), returns a `Signature` where you can get [`Signature.name`](https://docs.rs/git2/0.10.1/git2/struct.Signature.html#method.name) and [`Signature.email`](https://docs.rs/git2/0.10.1/git2/struct.Signature.html#method.email).

You can also get the time with [`Signature.when`](https://docs.rs/git2/0.10.1/git2/struct.Signature.html#method.when)